recipes/r-rmarkdown/meta.yaml

```diff
@@ -16,17 +16,17 @@ build:
 # Suggests: shiny (>= 0.11), testthat, digest
 requirements:
   build:
-    - r
+    - r-base
     - r-catools
-    - r-htmltools >=0.2.4
-    - r-knitr >=1.12.3
+    - r-htmltools >=0.3.5
+    - r-knitr >=1.14
     - r-yaml >=2.1.5
 
```

Comment:
@dyusuf Thanks for the skeleton suggestion, did not know about that. I updated the meta.yaml file according to the suggestions of the skeleton file.